android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -175,14 +175,32 @@ public void crash() {
     }
 
     @ReactMethod
-    public void fetchRelease(Promise promise) {
+    public void fetchNativeRelease(Promise promise) {
         WritableMap release = Arguments.createMap();
         release.putString("id", packageInfo.packageName);
         release.putString("version", packageInfo.versionName);
         release.putString("build", String.valueOf(packageInfo.versionCode));
         promise.resolve(release);
     }
 
+    @ReactMethod
+    public void fetchNativeAppStart(Promise promise) {
+        final AppStartState appStartInstance = AppStartState.getInstance();
+        final Date appStartTime = appStartInstance.getAppStartTime();
+
+        final double appStartTimestamp = (double) appStartTime.getTime();
```

Comment:
We need to keep in mind auto performance monitoring might be disabled or failed to run i the native layer. What's the return type here for `getTime`? Should we cast to `Double` instead assuming this can be null? I'm afraid it'll either throw on some invalid cast or give a `0` which could either break the math or be misleading

Reply:
`getTime` returns a `long`. So I will add the null check for `appStartTime` which can be null